commit 96e76e694101c2b09c997bce5f064df0d051b729
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Fri Oct 8 12:01:30 2010 -0700

    Staging: brcm80211: remove VALID_MASK macro
    
    No one is using it.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/bcmutils.h
+++ b/drivers/staging/brcm80211/include/bcmutils.h
@@ -451,7 +451,6 @@ extern "C" {
 
 #define CEIL(x, y)		(((x) + ((y)-1)) / (y))
 #define	ISPOWEROF2(x)		((((x)-1)&(x)) == 0)
-#define VALID_MASK(mask)	(!((mask) & ((mask) + 1)))
 
 /* bit map related macros */
 #ifndef setbit
